program/c/src/oracle/sort/tmpl/sort_stable.c

```diff
@@ -1,195 +0,0 @@
-/* Usage:
```

Comment:
you deleted this one because the sort is no longer stable (?)

(I don't think it matters that the sort is stable so that's fine)

Reply:
Yeah. even being sorted is not important.